src/client/app/graph/graph.component.ts

```diff
@@ -26,4 +28,53 @@ export class GraphComponent implements OnInit {
     });
   }
 
+  //TODO(eyuelt): this is a temporary hack for adding cards to canvas
+  addCardsAndArrows() {
+    //TODO(eyuelt): is first() the correct way to do this thing only once?
```

Comment:
I think so:
http://stackoverflow.com/questions/28007777/rxjs-create-subscribe-once-and-dispose-method